src/components/Tags/Events/EventStatus.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from 'react';
+import { Tag } from 'antd';
+import './eventStatus.css';
+
+function EventStatus(props) {
+  const [backgroundColor, setBackgroundColor] = useState();
+  const [fontColor, setFontColor] = useState();
+  const eventStatusHandler = () => {
+    if (props.label.toLowerCase() === 'published' || props.label.toLowerCase() === 'publié') {
```

Comment:
@AbhishekPAnil Please do not use localized labels when testing for conditions. There are enums "Published", "Draft" etc. that represent the state of the event independent of language. These enums are passed from the backend API and do not change with language. Otherwise, if you use the localized label in your conditions, and the labels in the i18n file changes, you will have a bug in your code.